src/base/compute/owl_computation_type.ml

```diff
@@ -54,37 +60,37 @@ module Make
     | Noop
     | Var
     | Const
-    | Empty                         of int array
-    | Zeros                         of int array
-    | Ones                          of int array
-    | Create                        of int array
-    | Sequential                    of int array
-    | Uniform                       of int array
-    | Gaussian                      of int array
-    | Bernoulli                     of int array
-    | Init                          of int array * (int -> elt)
-    | Get                           of int array
-    | Set                           of int array
-    | GetSlice                      of int list list
-    | SetSlice                      of int list list
+    | Empty of int array
```

Comment:
This one might be difficult, but do you think it is possible to keep the alignment of each item in `match`?

Reply:
so turns out this is a feature that they are considering adding in the future; it seems that it's not so straightforward to do  (see https://github.com/ocaml-ppx/ocamlformat/issues/360)

Reply:
hope this feature can be added soon, it looks better imo